### DISCUSSION TITLE
Vlan Group not remain assigned to interface

### DISCUSSION THREAD

**Discussion by davama** [netbox-community/netbox#8361] [Q&A]
Hello,

Had posted this on the wrong project... :disappointed: 

running (v3.1.5)

Our current hierarchy is as follows:

Site > vlan-group(s) > vlan_set_per_group(6 vlans)

When I assign an interface with my vlan_group, under `untagged_vlan` i am only able to see my 6 vlans, as expected.

After I save the changes, and go back, I see vlan_group as `----` empty.

This is how it looks assigned.
![link](https://i.imgur.com/eGAEDgZ.png)

After I save, and go back into the interface, i see it as so:
![link](https://i.imgur.com/BYyMvfs.png)

The behavior im looking for is that when i select the `Untagged VLAN` i only see the 6 vlans from the vlan_group. But with the current behavior, post save, i see all the vlans for the `Site`, which I dont want.

Why does the vlan_group not stay? Something im doing wrong?

Thank you,
Dave

**Comment by kkthxbye-code** [comment]
If I recall correctly, the VLAN group dropdown on prefix objects are purely used for filtering the Untagged VLAN drop down under it. There is no direct relation between prefixes and VLAN groups.

**Comment by davama** [comment]
@kkthxbye-code 
thank you for the input!
So then the vlan_group remaining assigned to the interface would fall under feature request?
